### PR TITLE
Bump shapeless to 2.2.5

### PR DESCRIPTION
**Pull request by triggerNZ**
We had a clash between maestro-core and one of coppersmith's dependencies, hence the bump

### CODE REVIEW

**Comment by stephanh** [comment]
Sorry, commented on the commit instead of the PR.

**Comment by triggerNZ** [comment]
I did see that, so I am a little concerned too. Let me explore a little more

File: uniform-dependency/src/main/scala/au/com/cba/omnia/uniform/dependency/UniformDependencyPlugin.scala [hunk 1]
```diff
@@ -220,7 +220,7 @@ object UniformDependencyPlugin extends Plugin {
       def specs         = "3.5"
       def scalaz        = "7.1.1"  // Needs to align with what is required by specs2
       def scalazStream  = "0.7a"   // Needs to align with what is required by specs2
-      def shapeless     = "2.1.0"  // Needs to align with what is required by specs2
+      def shapeless     = "2.2.5"  // Needs to align with what is required by specs2
```
Comment (stephanh) [inline uniform-dependency/src/main/scala/au/com/cba/omnia/uniform/dependency/UniformDependencyPlugin.scala]: Based on what we discussed can you remove the comment?

**Comment by stephanh** [comment]
:+1: once the comment is removed.

**Comment by stephanh** [comment]
Also can you bump the minor version number.

**Pull request merged**